lib/chef/knife/bootstrap.rb

```diff
@@ -375,9 +375,7 @@ def run
             exit 1
           end
 
-          client_builder.run
-
-          chef_vault_handler.run(node_name: config[:chef_node_name])
+          chef_vault_handler.run(client_builder.run[:client])
```

Comment:
Hmm. I think I like accessing via simple property over that. We'd want the methods to be `client!` and `node!` which feels a little odd to me in this case. I'm not "passionately" opposed to accessors that just return `nil` if run has not been called. Do you see issues with having them emitted from `run`?